pkg/controller/host/handler/vsphere/handler.go

```diff
@@ -49,9 +49,6 @@ func (r *Handler) Watch(watch *handler.WatchManager) (err error) {
 //
 // Resource created.
 func (r *Handler) Created(e libweb.Event) {
-	if !r.HasParity() {
-		return
-	}
```

Comment:
HasParity() no longer needs to be checked because the watch has been setup without the initial snapshot (list of model `Created` events).  The parity marker was used to _skip_ the initial `Created` events.